apps/stats/test/utils/hook-testing-utils.ts

```
@@ -0,0 +1,314 @@
+
+/**
+ * Test suite generators for common hook patterns
+ * These generate standard test cases that most hooks should have
+ */
+
+/**
+ * Generate standard parameter tests for hooks with range/order/id params
+ */
+export const generateParameterTests = (
+    hookName: string,
+    hookFunction: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)